lms/djangoapps/bulk_enroll/views.py

```diff
@@ -60,7 +60,6 @@ class BulkEnrollView(APIView):
     def post(self, request):
         serializer = BulkEnrollmentSerializer(data=request.data)
         if serializer.is_valid():
-            request.POST = request.data
```

Comment:
i see that self.request gets passed in to students_update_enrollment below, and that function accesses request.POST. however, i don't understand if self.request and request are the same object?